packages/rum-core/src/domain/view/trackViews.spec.ts

```diff
@@ -415,6 +412,7 @@ describe('view metrics', () => {
       }
       const { getViewUpdate, getViewUpdateCount, getViewCreateCount, startView } = viewTest
       startView()
+      clock.tick(0)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Is it because of the implementation of `waitAfterLoadEvent` ?

Reply:
Yes! `trackViews` tests depend on `trackNavigationTimings` logic, so this requirement "spreads". This is the case in quite a few unit test specs: we generally test more than we should, and I'll come up with a plan to reduce this in the feature.